src/v2/guide/typescript.md

```diff
@@ -70,7 +70,8 @@ const Component = {
 }
 ```
 
-To use vue types in environment without module bundler you can create custom type definition file containing declare global (see https://www.typescriptlang.org/docs/handbook/declaration-merging.html)
+To use Vue types in an environment without a module bundler, you can create a custom type definition file containing `declared global` (see [Declaration Merging
```

Comment:
The markup is broken here.

Reply:
Not sure I understood exactly what is broken in markup? Found a typo and added link to url label as well.

Reply:
Preview your markdown and you'll see what I mean.

Reply:
Interesting. I am probably missing something obvious because I did preview and looks correct to me. Here is what I am testing:

https://github.com/vuejs/vuejs.org/blob/6e6891eeb139c6dc712169359677e1bf72dd3c4b/src/v2/guide/typescript.md

And here is what I am seeing:

![image](https://user-images.githubusercontent.com/1261322/56531031-0104d000-655c-11e9-9bd8-b61f367fa504.png)